capstone/capapi/serializers.py

```diff
@@ -150,7 +150,9 @@ def as_dict(obj):
 
         # get extracted_citations list, removing duplicate c["cite"] values
         extracted_citations = []
-        for c in s["extracted_citations"]:
+        ec = [o['extracted_citations'] for o in s['casebody_data']['text']['opinions'] if 'extracted_citations' in o]
+        ec = [item for sublist in ec for item in sublist]
+        for c in ec:
```

Comment:
that info seems encoded in the blog anyways. will remove.

Reply:
this now pops the opinion_id.

Reply:
I mean, do we want to add  `"opinion_index": <index>-1` to `extracted_cite` in the next lines? (`-1` to account for moving `head_matter` around.) Otherwise there can be multiple copies of the same cite in the API with no way to tell why.

Reply:
looool, documents.py now includes the opinion id.

Reply:
src: https://github.com/harvard-lil/capstone/pull/1962/files#diff-464cbe1adc7d1bb94b3b7c4216797eef8f49312e8a58d5f6fd2a853e5842ef64R171